keyboards/cloud_macro/readme.md

```diff
@@ -0,0 +1,27 @@
+# cloud_macro
+
+![cloud_macro](imgur.com image replace me!)
```

Comment:
Can you provide an image here so that we can verify things like physical layout,